src/panels/config/entities/entity-aliases/dialog-entity-aliases.ts

```diff
@@ -133,6 +142,12 @@ class DialogEntityAliases extends LitElement {
     this._aliases[index] = (ev.target as any).value;
   }
 
+  private async _keyDownAlias(ev: KeyboardEvent) {
+    if (ev.key === "Enter") {
+      this._addAlias();
```

Comment:
Can t we do the `stopPropagation` here?

Reply:
I handled it at the container level because if user focus the delete button or the add alias button and press enter, we will have the same issue.
Or we can add the `stopPropagation` to each element ?

Reply:
OK, shouldn't we limit it to the enter key then?

Reply:
I replaced actions by buttons to avoid automatic submission. It's cleaner than stopping propagation.

Reply:
Not sure I agree... An update of the dialog now requires manual action here to adjust styling etc.

We have this issue in way more dialogs I noticed btw, maybe we should work on a more generic solution (in the future)